scripts/spotless.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: "com.diffplug.spotless"
+
+spotless {
+    encoding 'UTF-8'
+    kotlin {
+        target "**/*.kt"
+        ktlint()
+        trimTrailingWhitespace()
+        endWithNewline()
+    }
+    java {
+        importOrder()
+        removeUnusedImports()
+        googleJavaFormat()
+        trimTrailingWhitespace()
+        endWithNewline()
+    }
+}
```

Comment:
We don't have many `.kts` files, but we can consider adding rules for them. 😄 

```kotlin
  kotlinGradle {
    target '*.gradle.kts'
    ktlint()
  }
```

Reply:
DONE ;)